examples/specs/normalized/trellis_row_column_normalized.vl.json

```diff
@@ -23,6 +23,14 @@
       "y": {
         "field": "Miles_per_Gallon",
         "type": "quantitative"
+      },
+      "color": {
+        "field": "Cylinders",
+        "type": "nominal"
+      },
+      "shape": {
```

Comment:
I added it because I wanted to be sure that I got the order right. And I know once we implement https://github.com/vega/vega-lite/issues/2176, we will have to revisit this again. 

Yes, we need a different example for the docs but I'd say this one is not great anyway (too many cells).